corehq/apps/app_manager/static_strings.py

```diff
@@ -135,6 +137,7 @@
     ugettext_noop('Must correspond to the password format specified below.'),
     ugettext_noop('Native (International)'),
     ugettext_noop('Never'),
+    ugettext_noop('No'),
```

Comment:
You don't need this; it's a few lines down.

Reply:
I missed it.
Thanks for pointing out 😄